core/ecschema-metadata/src/SchemaKey.ts

```diff
@@ -19,6 +19,9 @@ export class ECVersion {
   private _write: number = 0;
   private _minor: number = 0;
 
+  // Define a static global instance representing zero
+  public static readonly ZERO = new ECVersion(0, 0, 0);
```

Comment:
Maybe it would be better to call this 'NoVersion' and document that a version with all zeros is an invalid version for a schema but using it in a key means no version has been specified.  As it stands this could mean anything and the docs don't help you figure what it should mean

Reply:
done. Do we want to prevent schemas to be constructed with a NO_VERSION version? I don't think we have validation on that end now...
I'll keep the comment open to remind me to add something to the docs